client/src/Ratings.jsx

```diff
@@ -82,8 +88,4 @@ class Ratings extends Component {
   }
 }
 
-Ratings.propTypes = {
-  reviews: PropTypes.any,
-};
-
 export default Ratings;
```

Comment:
I think ESLint requires a single line after the last line of code - to make it easier in the long run, try to follow the ESLint as you go to keep with the group styling

Reply:
Will do, thanks.